File: C/tabelline.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[])
{
    //Variabili
    int n;              //Var Input
    int counter = 0;    //Var output - lavoro
    int risultato;      //Var Output

    printf("Inserire un valore di ritorno per la tabellina: ");
    scanf("%d", &n);
    
    for (counter = 0; counter <= 10; counter++)     //Calcolatore di tabelline
    {
        risultato = n * counter;
        printf("%d x %d = %d\n", n, counter, risultato);
    }
}

File: JAVASCRIPT/tabelline.js
//Variabili
var n;              //Var in
var counter = 0;    //Var output -> counter - lavoro
var risultato;      //Var out

n = window.prompt('Inserire un valore di ritorno per la tabellina: ');
console.log('Tabellina del numero ' + n)

for (counter = 0; counter <= 10; counter++)
{
    risultato = n * counter;
    console.log(n + ' x ' + counter + ' = ' + risultato)
}